Demo/HWIFileDownload.xcodeproj/project.pbxproj

```diff
@@ -300,7 +322,7 @@
 					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
 				);
 				INFOPLIST_FILE = HWIFileDownload/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 6.0;
```

Comment:
Please try to keep `IPHONEOS_DEPLOYMENT_TARGET = 6.0`.

Reply:
@Heikowi I tried but Xcode 10 doesn't support 6 as deployment target. Even 7 or 8 isn't support anymore. Only iOS 9 up. 🤷‍♂️

Reply:
Have in mind that this is only for the Demo. The library still works on older iOS versions.